src/components/Checkout/Checkout.jsx

```diff
@@ -11,94 +10,150 @@ import amex from '../../assets/images/american-express.png';
 import paypal from '../../assets/images/paypal.png';
 import visa from '../../assets/images/visa.png';
 import stripe from '../../assets/images/stripe.png';
+import api from '../../configs/api';
+import toast from 'react-hot-toast';
+import { getAxiosErrorMessage } from '../../utils';
 
-const initialFormData = {
-  orderItems: [],
-  paymentMethod: 'Credit Card',
-  shippingAddress: {
-    address: '',
-    city: '',
-    state: '',
-    postcode: '',
-    phone: ''
-  }
-};
-
-const confirmOrder = async (formData) => {
+// Function to place an order
+const placeOrder = async (orderData) => {
   try {
-    const response = await axios.post('http://localhost:500/api/order', formData);
+    // Retrieve the user's token from local storage
+    const token = localStorage.getItem('token');
+    // Send a POST request to create an order
+    const response = await api.post('/orders', orderData, {
+      headers: {
+        Authorization: `Bearer ${token}`
```

Comment:
You don't need to add the token here as api config is already handling it for us.

Reply:
I didn't have it in the first place and got the authentication error so I added in, but I'll try remove and test again to see if the issue is still there.